plugins/doc_fragments/onepassword.py

```diff
@@ -0,0 +1,60 @@
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2023, Ansible Project
+# GNU General Public License v3.0+ (see LICENSES/GPL-3.0-or-later.txt or https://www.gnu.org/licenses/gpl-3.0.txt)
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+
+class ModuleDocFragment(object):
+    DOCUMENTATION = r'''
+requirements:
+  - See U(https://support.1password.com/command-line/)
+options:
+  _terms:
+    description: Identifier(s) (UUID, name, or subdomain; case-insensitive) of item(s) to retrieve.
+    required: true
+  master_password:
+    description: The password used to unlock the specified vault.
+    aliases: ['vault_password']
+  section:
+    description: Item section containing the field to retrieve (case-insensitive). If absent will return first match from any section.
+  domain:
+    description: Domain of 1Password.
+    version_added: 3.2.0
+    default: '1password.com'
+    type: str
+  subdomain:
+    description: The 1Password subdomain to authenticate against.
+  account_id:
+    description: The account ID to target.
+    type: str
+    version_added: 7.5.0
+  username:
+    description: The username used to sign in.
+  secret_key:
+    description: The secret key used when performing an initial sign in.
+  service_account_token:
+    description:
+      - The access key for a service account.
+      - Only works with 1Password CLI version 2 or later.
+    type: str
+    version_added: 7.1.0
+  vault:
+    description: Vault containing the item to retrieve (case-insensitive). If absent will search all vaults.
+notes:
+  - This lookup will use an existing 1Password session if one exists. If not, and you have already
+    performed an initial sign in (meaning C(~/.op/config), C(~/.config/op/config) or C(~/.config/.op/config) exists), then only the
+    C(master_password) is required. You may optionally specify O(subdomain) in this scenario, otherwise the last used subdomain will be used by C(op).
```

Comment:
```suggestion
    O(master_password) is required. You may optionally specify O(subdomain) in this scenario, otherwise the last used subdomain will be used by C(op).
```